recipe-server/client/control/selectors/FiltersSelector.js

```diff
@@ -132,7 +132,7 @@ export const isFilteringActive = state =>
  * Get all filters as an object keyed on their slug
  *
  * @param  {Array<Object>} groups All filter groups
```

Comment:
This still has the wrong parameter name.

Also, I'm still curious: Why change the parameter name to `list` instead of `groups`?